client/webserver/site/src/html/register.tmpl

```diff
@@ -89,6 +89,19 @@
       </div>
     </form>
 
+    {{- /* registration failed low balance. */ -}}
+    <form class="card mx-auto my-5 bg1 d-hide" id="failedRegForm">
+      <div class="bg2 px-2 py-1 text-center fs18">Insuficient funds</div>
+      <div class="p-4">
+        <div class="fs16">
+          Looks like you do not have enough funds for paying the fee in the
+          selected account. Please fund the account and try again.
+        </div>
+        <div class="fs15 pt-3 text-center d-hide errcolor" id="regFundsErr"></div>
+        <hr class="dashed mt-4">
+      </div>
```

Comment:
This is still just a guess that it's insufficient funds, but really it's any error from `wallet.PayFee`, which returns a `feeSendErr`-coded error.

It could be many things including a bad fee address, or an error from `sendWithReturn` such as failure to sign the tx or failure to broadcast with sendrawtransaction.

Thus, I think it's warranted to soften the language in this dialog to indicate an error, but suggest the most likely cause of insufficient funds, and ofc show the whole error message as you are doing.